src/lucky/text_response.cr

```diff
@@ -42,4 +43,19 @@ class Lucky::TextResponse < Lucky::Response
         Lucky::Server.settings.gzip_content_types.includes?(content_type)
     {% end %}
   end
+
+  private def write_cookies : Void
+    context.cookies.set(
```

Comment:
Can we still break this one out in to a `write_session` method?

Reply:
To be honest, I have been doubting about keeping them separate as well. My feeling was we're "writing cookies" here so I merged the two methods. But I can split them out again if you prefer.

Reply:
They do sort of go hand in hand, but my thought is they're still treated and accessed differently. Plus it makes it easier to quickly look at the code and see what is doing what.

Reply:
You're right, it makes for an easier red. I'll revert that back.